react/test/setup.js

```diff
@@ -1,10 +1,12 @@
 const { JSDOM } = require("jsdom");
+const channels = require("worker_threads");

```

Comment:
it looks like this isn't being used anywhere. is it really necessary? :smile:

Reply:
Its being used below, it is necessary for node <= 14 to work with react testing library with jsdom